programs/btc_spv_api/src/spv_processor.rs

```diff
@@ -163,4 +164,33 @@ mod test {
 
         Ok(())
     }
+
+    #[test]
+    fn test_parse_transaction_hex() {
+        let testblockhash = "0000000000000bae09a7a393a8acded75aa67e46cb81f7acaa5ad94f9eacd103";
+        let testtxhash = "5b09bbb8d3cb2f8d4edbcf30664419fb7c9deaeeb1f62cb432e7741c80dbe5ba";
+
+        let mut testdatabytes = include_bytes!("testblock.in");
+        let mut headerbytes = hex::encode(&testdatabytes[0..]);
+        let hbc = &headerbytes[0..80];
+
+        let i: usize = 5;
+
+        let mut txdata = &testdatabytes[80..];
+
+        let vilen = measure_variable_int(&txdata[0..9]).unwrap();
+        let txnum = decode_variable_int(&txdata[0..9]).unwrap();
+
+        txdata = &txdata[vilen..];
+        let tx = Transaction::new(txdata.to_vec());
+        println!("tx_num: {}", txnum);
```

Comment:
@mvines, this snuck by nits.sh. A bug?

Reply:
println! free is currently opt-in: https://github.com/solana-labs/solana/blob/86213d38fe66aa5aaa359d2daca53b5ed42274e9/ci/nits.sh#L20

Maybe we should flip that around

Reply:
Yeah, opt-out seems better.  I'll add an issue.  Very low priority.

Reply:
👍 cc: #5961